tensorflow/python/ops/image_ops_test.py

```diff
@@ -6288,6 +6288,16 @@ def testImageCropAndResizeWithInvalidInput(self):
             crop_size=[2065374891, 1145309325])
         self.evaluate(op)
 
+  def testImageCropAndResizeWithNon1DBoxes(self):
+    with self.session():
+      with self.assertRaises((errors.InvalidArgumentError, ValueError)):
```

Comment:
Can you `assertRaisesRegex`, and ensure the error message between graph/eager mode is somewhat consistent?